docs/source/en/index.mdx

```diff
@@ -212,6 +213,7 @@ Flax), PyTorch, and/or TensorFlow.
 |           CANINE            |       ✅       |       ❌       |       ✅        |         ❌         |      ❌      |
 |            CLIP             |       ✅       |       ✅       |       ✅        |         ✅         |      ✅      |
 |           CodeGen           |       ✅       |       ✅       |       ✅        |         ❌         |      ❌      |
+|      conditional_detr       |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
```

Comment:
```suggestion
|      conditional_detr       |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
```
Normally this will also be updated to Conditional DETR when running `make fix-copies` again from the root of the repo

Reply:
It is a little bit weird, I just ran the `make fix-copies`, it seems that nothing changed.

Reply:
Seems like this is fine now!